content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -45,34 +45,32 @@ The following prerequisites are needed in order to utilize IPv4/IPv6 dual-stack
    * Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
    * A network plugin that supports dual-stack (such as Kubenet or Calico)
 
-## Enable IPv4/IPv6 dual-stack
+## Configure IPv4/IPv6 dual-stack
 
-To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for the relevant components of your cluster, and set dual-stack cluster network assignments:
+To configure IPv4/IPv6 dual-stack, set dual-stack cluster network assignments:
```

Comment:
When features are beta, we typically still mention that the feature gate needs to be enabled, but tweak the wording. This helps people who are using older versions of Kubernetes spot that they might want to swap to reading documentation appropriate for their version.